src/lib/drupal/facilitySideNav.test.ts

```diff
@@ -14,9 +23,19 @@ const mockMenu: Partial<Menu> = {
       enabled: true,
       field_menu_section: 'Test Section',
       items: [], // Assuming it's a flat structure for simplicity
+      menu_name: 'test-menu',
+      provider: 'menu_link_content',
+      weight: '0',
+      options: {},
+      route: {
+        name: 'entity.node.canonical',
+        parameters: {},
+      },
+      type: 'menu_link_content',
+      meta: {},
+      parent: '',
     },
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ] as any[], // Use 'any[]' to bypass detailed type requirements
+  ] as CustomMenuLinkContent[],
```

Comment:
I got a comment from Chris about avoiding type assertions...but I dunno how to avoid that with this partial menu and custom type. Maybe Chris has an opinion on this?